src/components/PersonLink.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+};
+
+const FEMALE = 'f';
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
```

Comment:
```suggestion
export const PersonItem: React.FC<Props> = ({ person }) => {
```

This component is not link